app/dbsetup.py

```diff
@@ -50,14 +50,6 @@ class RoutingSQLAlchemy(SQLAlchemy):
     def on_reader(self):
         return self.session().using_bind("reader")
 
-    def apply_driver_hacks(self, app, info, options):
-        super().apply_driver_hacks(app, info, options)
-        if "connect_args" not in options:
-            options["connect_args"] = {}
-        options["connect_args"]["options"] = "-c statement_timeout={}".format(
```

Comment:
I'm wondering if we could add a flag to specify proxy is used. For local development would we not still want the timeout ?